org.midori_browser.Midori.json

```diff
@@ -0,0 +1,50 @@
+{
+  "app-id": "org.midori_browser.Midori",
+  "branch": "stable",
+  "runtime": "org.gnome.Platform",
+  "runtime-version" : "3.28",
```

Comment:
Why `3.28`? Doesn't it build with latest SDK, `3.32`?

Reply:
it was an oversight.